From 701d6127178f3e5a0a55a0e2cbfe9894bdeaca7a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:29 -0500
Subject: [redhat] kabi: Adding symbol pnp_get_resource (drivers/pnp/resource.c)

Message-id: <34a15d4afd7e9a9ed9f3d27d4fbfd30f21ab0229.1418903403.git.poros@redhat.com>
Patchwork-id: 102276
O-Subject: [PATCH RHEL7.1 37/48] kabi: Adding symbol pnp_get_resource (drivers/pnp/resource.c)
Bugzilla: 1160754
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160754
    https://bugzilla.redhat.com/show_bug.cgi?id=1160754

    Description:
    ============
    The large chuncks of memory that are reserved to be superpages
    are defined as reserved to the BIOS and passed to the the superpages
    module as an ACPI device, "superpages". This symbol is on of several
    required to handle that device.
    Used in superpages.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1062669 49/49] kabi: Adding symbol pnp_get_resource (drivers/pnp/resource.c)
 Jiri Olsa <jolsa@redhat.com> (Fri, 14 Feb 2014 14:24:11 +0100)
 Description:
 ============
 The large chuncks of memory that are reserved to be superpages
 are defined as reserved to the BIOS and passed to the the superpages
 module as an ACPI device, "superpages".   These symbols are
 required to handle that device.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pnp_get_resource

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pnp_get_resource b/redhat/kabi/kabi-module/kabi_x86_64/pnp_get_resource
new file mode 100644
index 0000000..210e260
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pnp_get_resource
@@ -0,0 +1,2 @@
+#1-
+0xf4d63ca3 pnp_get_resource vmlinux EXPORT_SYMBOL
-- 
1.7.1